packages/lib/src/components/Card/Card.tsx

```diff
@@ -33,7 +33,7 @@ export class CardElement extends UIElement<CardElementProps> {
             hasCVC: !((props.brand && props.brand === 'bcmc') || props.hideCVC),
             // billingAddressRequired only available for non-stored cards
             billingAddressRequired: props.storedPaymentMethodId ? false : props.billingAddressRequired,
-            ...(props.brands && !props.groupTypes && { groupTypes: props.brands }),
+            // ...(props.brands && !props.groupTypes && { groupTypes: props.brands }),
```

Comment:
Can we delete this comment? Commented code might cause confusion in the future

Reply:
Funny - I'm not seeing this in my file...

Reply:
I'll try recommitting it